src/components/AdditionalInformationComponents/index.js

```diff
@@ -130,11 +130,25 @@ const AdditionalInformationComponent = props => {
   const validateAdditionalInputsAndSignIn = () => {
     if (validateInputs()) {
       signUp({
-        pageChange: () => history.push(process.env.REACT_APP_MAIN_URL),
-        setFailFeedbackMsg: () => {},
         id: id.value,
         nickName: nickName.value,
         password: password.value
+      }).then(response => {
+        if (!response) {
+          id.setFeedbackMsgAndValidation("회원가입에 실패했습니다", false);
+          nickName.setFeedbackMsgAndValidation(
+            "회원가입에 실패했습니다",
+            false
+          );
+          password.setFeedbackMsgAndValidation(
+            "회원가입에 실패했습니다",
+            false
+          );
+          passwordConfirm.setFeedbackMsgAndValidation(
+            "회원가입에 실패했습니다",
+            false
+          );
+        }
```

Comment:
각 컴포넌트에 똑같은 메세지를 날리는 건가요?! 
개인적인 소견으로 한번의 메세지만 보내면 좋을 거란 생각이 듭니다 ㅎㅎ 
여기서는 이렇게 중복을 줄일 수 있을 것 같아요 ㅎㅎ 

```suggestion
          [nickName, password, passwordConfirm].forEach(el =>
                  el.setFeedbackMsgAndValidation(
                      "회원가입에 실패했습니다",
                      false
          );
```